### DISCUSSION TITLE
Why install av twice?

### DISCUSSION THREAD

**Discussion by zhongqiu1245** [open-mmlab/mmediting#1855] [General]
Hello dear authors!
Thank you for your amazing job!
In mmagic/requirements/runtime.txt, I find mmagic installs av twice:
```
av
av==8.0.3; python_version < '3.7'
```
Could you tell me why mmagic installs av twice? 
And the python_version must be lower than 3.7? Because other mm-products require python>=3.8. It will have negative effect  for mmagic if I install mmagic in python>=3.8?
Thank you !

**Comment by Z-Fran** [comment]
this mean we install av==8.0.3 when python_version < '3.7'. MMagic can be used on python>=3.8 .

**Reply by zhongqiu1245** [reply]
thx!
